config/common/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml

```diff
@@ -41,7 +41,7 @@ enable_nodelocaldns: false
 
 kubernetes_audit: true
 audit_log_path: "/var/log/audit/kube-apiserver-audit.log"
-audit_log_maxage: 7
+audit_log_maxage: 30
```

Comment:
Do we really need this to be 30 since we're anyway storing the logs in s3

Reply:
Sure, but if we know that we fulfill that anyway, then I think that we can leave it at 7. As long as we document the reason we have left it like that.